src/modules/projects/hooks/useProjectDashboardNavItems.tsx

```diff
@@ -90,23 +98,10 @@ export const useProjectDashboardNavItems = (
             hide: !project.serviceTypes.find((s) => s.supportsBulkAssignment),
           },
           {
-            // Legacy Referral page. Hidden if CE is enabled for the project.
             id: 'referrals',
             title: 'Referrals',
-            path: ProjectDashboardRoutes.REFERRALS,
-            permissions: [
-              'canManageIncomingReferrals',
-              'canManageOutgoingReferrals',
-            ],
-            permissionMode: 'any',
-            hide: !!project.coordinatedEntryFeatures,
-          },
-          {
-            id: 'ce-referrals',
-            title: 'Referrals',
             path: ProjectDashboardRoutes.CE,
-            hide:
-              !showReferrals && !showAvailableUnits && !showOutgoingReferrals,
+            hide: !ceReferralsTabEnabled && !legacyReferralTabEnabled,
```

Comment:
Here we are removing the 2 separate "Referrals" pages, in favor of always using the Project Referrals page if any referral functionality is visible, whether CE or legacy. Is that right? Sounds good to me 👍 

I think this is functionally equivalent:
```suggestion
            hide: !(showReferrals || showAvailableUnits || showOutgoingReferrals || showLegacyReferrals)
```
and I find it easier to read, rather than introducing variable names that refer to "tabs". (Especially because the name `ceReferralsTabEnabled` is a bit misleading, as there are multiple different tabs within the Project Referrals page that its constituent values `showReferrals || showAvailableUnits || showOutgoingReferrals` refer to)

Reply:
sounds good, updated!